### PR TITLE
support configuring protocols

### PR DESCRIPTION
**Pull request by yanns**
@hajoeichler @emmenko please review and new release please

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,16 +62,18 @@ General command line options can be seen by simply executing the command `node l
...
-    --debug                      give as many feedback as possible
+    -h, --help                       output usage information
+    -V, --version                    output the version number
+    -p, --projectKey <key>           your SPHERE.IO project-key
+    -i, --clientId <id>              your OAuth client id for the SPHERE.IO API
+    -s, --clientSecret <secret>      your OAuth client secret for the SPHERE.IO API
+    --sphereHost <host>              SPHERE.IO API host to connect to
+    --sphereProtocol <protocol>      SPHERE.IO API protocol to connect to
+    --sphereAuthHost <host>          SPHERE.IO OAuth host to connect to
+    --sphereAuthProtocol <protocol   SPHERE.IO OAuth protocol to connect to
```
Comment (hajoeichler) [inline README.md]: `>` missing

File: src/coffee/run.coffee [hunk 2]
```diff
@@ -91,7 +91,9 @@ module.exports = class
       .option '-s, --clientSecret <secret>', 'your OAuth client secret for the SPHERE.IO API'
       .option '--accessToken <token>', 'an OAuth access token for the SPHERE.IO API, used instead of clientId and clientSecret'
       .option '--sphereHost <host>', 'SPHERE.IO API host to connect to'
+      .option '--sphereProtocol <protocol>', 'SPHERE.IO API protocol to connect to'
```
Comment (hajoeichler) [inline src/coffee/run.coffee]: also here, should we add the default.
`, 'https'`

File: README.md (hunk 1 above)
Comment (yanns) [inline README.md]: good catch! Fixed.

File: src/coffee/run.coffee (hunk 2 above)
Comment (yanns) [inline src/coffee/run.coffee]: then we do the same for `sphereHost`, `sphereAuthHost` and `sphereAuthHost`?
I am not for it.

Comment (emmenko) [inline src/coffee/run.coffee]: The sdk has defaults anyways...

Comment (yanns) [inline src/coffee/run.coffee]: Yes I prefer to let the defaults in the SDK and not to repeat them everywhere.

Comment (emmenko) [inline src/coffee/run.coffee]: :+1:

Comment (hajoeichler) [inline src/coffee/run.coffee]: ok, makes sense!

**Pull request merged**